catppuccin-mocha.theme

```diff
@@ -1,11 +0,0 @@
-base: "DARK"
-colors:
-  accent: "#fff5c2e7"           # pink
-  accentVariant: "#ffcba6f7"   # mauve
-  secondary: "#fff5c2e7"        # pink
-  secondaryVariant: "#ffcba6f7" # mauve
-  background: "#ff1e1e2e"       # base
-  menus: "#ff313244"            # surface0
-  title: "#ffb4befe"            # lavender
```

Comment:
I'm curious why all of these hex codes are prefixed with `ff`? Surprised it works like this. For example the `accent` color here is `#fff5c2e7`, a light yellow, when it should be `#f5c2e7`, a magenta...